composer.json

```diff
@@ -9,14 +9,14 @@
         "email" : "henrik@bjrnskov.dk"
     }],
     "require": {
-        "php"                      : ">=5.3.3",
-        "symfony/framework-bundle" : "^2.7 || ^3.0",
-        "symfony/console"          : "^2.7 || ^3.0",
+        "php"                      : "^5.4 || ^7.0",
+        "symfony/framework-bundle" : "^2.7 || ^3.0 || ^4.0",
+        "symfony/console"          : "^2.7 || ^3.0 || ^4.0",
         "henrikbjorn/stampie"      : "~0.11"
     },
     "require-dev": {
         "stof/stampie-extra": "~0.2@dev",
-        "symfony/phpunit-bridge": "^3.2"
+        "symfony/phpunit-bridge": "^3.4 || ^4.0"
```

Comment:
you could just use `^4.0` here, as this bridge supports all PHP versions even in 4.x